### DISCUSSION TITLE
How to setup this golang server.

### DISCUSSION THREAD

**Discussion by lc19997** [Junjie529/Golang-server-gin-#15] [Q&A] [answered]
I can not install this golang server.
Please give me a detailed guide.

**Comment by outdoorSpirit** [comment, marked as answer]
First, install docker.
Second, deploy server part of docker.
And  go run main.go

**Comment by outdoorSpirit** [comment]
First, go to ruby signal generator.
export SERVER_ENDPOINT=http://localhost:31333
export BEARER_TOKEN=EOtd08fod-EKxV8PU7jyNw
export DEMO_MODE=true

bundle exec ruby main.rb
